### PR TITLE
developer's mode

### PR DESCRIPTION
**Pull request by dbusel**

### CODE REVIEW

File: src/server/services/comsatService.js [hunk 1]
```diff
@@ -187,9 +191,10 @@ const closePortOnComsat = function (params, callback) {
     });
     console.log(data);
 
+    // TODO: add dev mode
```
Comment (alexandershpak) [inline src/server/services/comsatService.js]: del

File: src/server/services/comsatService.js [hunk 2]
```diff
@@ -66,9 +66,10 @@ const openPortsOnComsat = function (params, callback) {
             '"heartbeatabsencethresholdport1":200000, "heartbeatabsencethresholdport2":200000}'
         });
 
+    // TODO: add dev mode
```
Comment (alexandershpak) [inline src/server/services/comsatService.js]: del

File: src/server/services/comsatService.js [hunk 3]
```diff
@@ -126,9 +128,10 @@ const closePortsOnComsat = function (params, callback) {
             });
             console.log(data);
 
+            // TODO: add dev mode
```
Comment (alexandershpak) [inline src/server/services/comsatService.js]: del

File: src/server/services/comsatService.js [hunk 4]
```diff
@@ -284,10 +290,11 @@ const verifyComsatConnections = function (params, callback) {
             process.stdout.cursorTo(0);
             percentage_done = Math.round((count / params.satellite.length) * 100);
 
+            // TODO: add dev mode
```
Comment (alexandershpak) [inline src/server/services/comsatService.js]: del

File: src/server/services/comsatService.js [hunk 5]
```diff
@@ -65,10 +65,10 @@ const openPortsOnComsat = function (params, callback) {
             mapping: '{"type":"private","maxconnectionsport1":1, "maxconnectionsport2":1, ' +
             '"heartbeatabsencethresholdport1":200000, "heartbeatabsencethresholdport2":200000}'
         });
-
+    
     let options = {
         host: params.satellite.domain,
-        port: 443,
+        port: 8080,
```
Comment (epankou) [inline src/server/services/comsatService.js]: How about moving this hardcoded values out to config?

File: src/server/services/comsatService.js [hunk 6]
```diff
@@ -80,6 +80,7 @@ const openPortsOnComsat = function (params, callback) {
     if (params.satellite.cert && params.satellite.selfSignedCerts === true) {
         let ca = '-----BEGIN CERTIFICATE-----\n' + params.satellite.cert + '\n' + '-----END CERTIFICATE-----';
         options.ca = new Buffer(ca);
+        options.port = 443;
```
Comment (epankou) [inline src/server/services/comsatService.js]: How about moving this hardcoded values out to config?

**Pull request merged**